### DISCUSSION TITLE
Adding images to the about.md

### DISCUSSION THREAD

**Discussion by Shamik-07** [chrede88/qubt#175] [Q&A] [answered]
Thank you very much for the theme. 
Can you please tell me with the current folder structure of the theme template, how can i add an image to the about page ?

**Comment by chrede88** [comment, marked as answer]
You can add an image using the standard markdown image embed method: `![Some image text](path/relative/to/assets)`.

You've to put the following into your `hugo.yaml` configuration file:
```yaml
markup:
  goldmark:
    renderHooks:
      image:
        enableDefault: true
```

Let's say you've an image called `my-image.png` in the `assets` folder. Then you can add the image to the about section by adding `![My image](my-image.png)` to the text part of `about.md`:

```markdown
---
# don't change
type: misc

# change these
jobTitle: Blogger
name: John Doe
bio: true
icon: ":coffee:"
---

Lorem ipsum dolor sit amet, bla bla ...

![About image](my-image.png)
```

**Reply by Shamik-07** [reply]
Thank you very much for answering. 

I added the image under the `assets` folder and it worked, but when the image was in the same directory level of `about.md` it wasn't working.

**Reply by chrede88** [reply]
This is the way the default markdown image embedding works for Hugo.
